deploy/helm/cf-operator/values.yaml

```diff
@@ -1,41 +1,86 @@
-# Default values for cf-operator.
+## Default values for Cf-operator Helm Chart.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+
+# applyCRD is a boolean to control the installation of CRD's.
+applyCRD: true
+
+cluster:
+  # domain is the the Kubernetes cluster domain
+  domain: "cluster.local"
+
+# fullnameOverride overrides the release name
+fullnameOverride: ""
+
+# image is the docker image of quarks job.
 image:
+  # repository that provides the operator docker image.
   repository: cf-operator
+  # org that provides the operator docker image.
   org: cfcontainerization
-  tag: foobar
+  # tag of the operator docker image
+  tag: v1.0.0-2.g4ceba93a
```

Comment:
We had a story some time ago, to explicitly set this to an invalid tag. Can you change it back to foobar? :)

Reply:
Done.